types/test.ts

```diff
@@ -192,4 +192,43 @@ switch(allthings.type) {
     break;
   default:
     break;
-}
\ No newline at end of file
+}
+
+// Test exception constructor with just message.
```

Comment:
I know that it is not done this way anywhere yet, but I would wrap this test into a function. The important thing is not that it is called. The test is just used to by the TypeScript-validator.

It is not executed.

Just make sure that the main use-cases (creation with message and node, access to fields) are covered.
The tests are currently more verbose than they need to be. It is not important that `Just a message` is really this string. Assigments like the following are enough:

```typescript
function testExceptionTypings() {
   let exception: Handlebars.Exception = new Handlebars.Exception("message", {
      type: 'MustacheStatement',
      ...
   })
   let message: string = exception.message
   let lineNumber: number = exception.lineNumber
   ...
}
```